src/android/android-backend.h

```diff
@@ -0,0 +1,37 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "../backend.h"
+
+namespace librealuvc
+{
+    namespace platform
+    {
+        class android_backend : public std::enable_shared_from_this<android_backend>, public backend
+        {
+        public:
+            android_backend() = default;
+            ~android_backend() = default;
+
+            std::shared_ptr<uvc_device> create_uvc_device(uvc_device_info info) const override;
+            std::vector<uvc_device_info> query_uvc_devices() const override;
+
+
+            // don't change
```

Comment:
I think these "don't change" comments were development notes and I don't really know what they mean here. Little bit misleading as well because both `create_usb_device` and `query_usb_devices` are also unsupported. I think these comments ought to be revised.

Reply:
Yeah, they don't make much sense to me either. I am not quite sure what we can do with them other than remove them :/ So I've changed some of them in 7b23b09ee